test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -405,6 +405,113 @@ namespace iroha {
       CHECK_ERROR_CODE_AND_MESSAGE(cmd_result, 2, query_args);
     }
 
+    class RemovePeer : public CommandExecutorTest {
+     public:
+      void SetUp() override {
+        CommandExecutorTest::SetUp();
+        peer = makePeer("address",
+                        shared_model::interface::types::PubkeyType{"pubkey"});
+        another_peer = makePeer(
+            "another_address",
+            shared_model::interface::types::PubkeyType{"another_pubkey"});
+        createDefaultRole();
+        createDefaultDomain();
+        createDefaultAccount();
+        CHECK_SUCCESSFUL_RESULT(
+            execute(*mock_command_factory->constructAddPeer(*peer), true));
+      }
+
+      std::unique_ptr<MockPeer> peer, another_peer;
+    };
+
+    /**
+     * @given command
+     * @when trying to remove peer
+     * @then peer is successfully removed
+     */
+    TEST_F(RemovePeer, Valid) {
+      addAllPerms();
+      auto new_peer =
+          makePeer("new_address",
```

Comment:
you create `another_peer` member in `class RemovePeer`, why not use it? here and forth

Reply:
also the code to add another peer could be moved to member func & reused